test/unit/tag_test.rb

```diff
@@ -294,4 +294,9 @@ def setup
     assert lat.location_tag?
     assert lon.location_tag?
   end
+
+  test 'Tag.get_recommendations' do
+    nodes = Tag.get_recommendations("blog", "test")
+    assert_not_nil nodes
```

Comment:
Here we are asserting that we are returning SOMETHING. Next we should be sure that the model code for `get_recommendations` actually does return something, and if not, debug that. Does that make sense?

Reply:
Now that I added this test, we are actually seeing it return success (not failure!) in the test logs. You can click on "Details" for Unit tests below to see the output. Here's a deep link:

https://github.com/publiclab/plots2/runs/8193928002?check_suite_focus=true#step:5:90

What I wonder though, is if we are getting [] back -- and i just checked, `[].nil?` evaluates to `false` in ruby! So, we may need to test `nodes.length > 0` to be sure SOMETHING is returned and not an empty array.

Reply:
Let's try!

```suggestion
    assert_not_nil nodes
    assert nodes.length > 0
```